Slam/ShowQueueViewController.swift

```diff
@@ -24,6 +27,22 @@ public class ShowQueueViewController: UIViewController {
         queueManager!.loadQueue()
     }
 
+
+    @IBAction public func clickRefreshQueue(sender: AnyObject) {
```

Comment:
I might just call this `refreshQueue`, since it could theoretically be triggered from a pull down, a click, etc.